recipes/fiji-max-inscribed_circles/meta.yaml

```diff
@@ -22,7 +22,15 @@ test:
     - test.groovy
 
 about:
-  home: https://imagej.net/Max_Inscribed_Circles
-  # TODO
-  license: GPL3
-  summary: This is an implementation of the Largest Inscribed Circle algorithm using an euclidean distance map.
```

Comment:
> This is an implementation of an iterative Largest Inscribed Circle algorithm using a euclidean distance map

Reply:
I put: `summary: ImageJ / Fiji plugin implementing an iterative Largest Inscribed Circle algorithm using a euclidean distance map`

Reply:
Is it ok?